pkgs/top-level/static.nix

```diff
@@ -77,35 +77,251 @@ self: super: let
       });
     };
 
-in {
+in rec {
```

Comment:
this file does was removed on staging next, please move this to the main derivations and rebase.